.upstream-tests/test/cuda/atomics/atomic.ext/atomic_fetch_max.pass.cpp

```diff
@@ -60,14 +60,58 @@ struct TestFn {
   }
 };
 
+template <template<typename, typename> typename Selector, cuda::thread_scope ThreadScope>
+struct TestFn<int, Selector, ThreadScope> {
```

Comment:
Specifically, I'm attempting to get a guarantee that signed math is working as expected. It would be completely fair to split it into unsigned and signed specializations. Perhaps more tests for this API are needed. ;)